packages/core/core/src/fs.ts

```diff
@@ -0,0 +1,93 @@
+/* global module process require */
+/* tslint:disable */
+import { joinPath } from 'embark-utils';
+import * as fs from 'fs-extra';
+import parseJson from 'parse-json';
+import * as path from 'path';
+import 'colors';
+
+export function mkdirpSync(args: any) { return fs.mkdirpSync(args); }
+
+export function mkdirp(args: any, args2?: any) { return fs.mkdirp(args, args2); }
+
+export function readdir(args: any, args2?: any) { return fs.readdir(args, args2); }
+
+export function stat(args: any, args2?: any) { return fs.stat(args, args2); }
+
+export function remove(args: any, args2: any) { return fs.remove(args, args2); }
+
+export function copy(args: any, args2?: any, args3?: any, args4?: any) { return fs.copy(args, args2, args3, args4); }
+
+export function copySync(args: any, args2?: any, args3?: any) { return fs.copySync(args, args2, args3); }
+
+export function move(args: any, args2?: any, args3?: any, args4?: any) { return fs.move(args, args2, args3, args4); }
+
+export function moveSync(args: any, args2?: any, args3?: any) { return fs.moveSync(args, args2, args3); }
+
+export function symlink(args: any, args2?: any, args3?: any, args4?: any) { return fs.symlink(args, args2, args3, args4); }
+
+export function appendFileSync(args: any, args2: any, args3?: any) { return fs.appendFileSync(args, args2, args3); }
+
+export function writeFile(args: any, args2: any, args3?: any, args4?: any) { return fs.writeFile(args, args2, args3, args4); }
+
+export function writeFileSync(args: any, args2: any, args3?: any) { return fs.writeFileSync(args, args2, args3); }
+
+export function readFile(args: any, args2?: any, args3?: any) { return fs.readFile(args, args2, args3); }
+
+export function readFileSync(args: any, args2?: any) { return fs.readFileSync(args, args2); }
+
+export function readdirSync(args: any, args2?: any) { return fs.readdirSync(args, args2); }
+
+export function statSync(args: any, args2?: any) { return fs.statSync(args, args2); }
+
+export function readJSONSync(args: any, args2?: any) {
```

Comment:
fs-extra provides a [`readJSONSync`](https://github.com/jprichardson/node-fs-extra/blob/master/lib/json/index.js#L14) function.

I don't recall if there was a special reason we implement it here instead of using fs-extra's implementation. It may be that at the time we were using an older version of fs-extra that didn't have that function, not really sure (cc: @iurimatias, @jrainville — any ideas?).

If there isn't a good reason for having our own, I think we should forward calls to fs-extra's function instead. Also, in the case the caller wants the behavior where exceptions are caught, an error message is logged and the process exits, I think that should be the responsibility of the caller or provided by a util function — it seems like a bad idea for that behavior to be in the `fs` function directly (though I may have been the one that added it back in the day, I sort of recall doing that).

Reply:
I agree. This module should be revisited once the move to `embark-core` was successful.

Reply:
No idea. I agree that we use fs-extra